examples/tch-ci/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
```

Comment:
Not sure we want to keep this file. This will likely become out of date as we are not used to do this. You can remove it.

Reply:
I may misread the issue. Sorry. If so, please just close this PR.

I guess [the issue](https://github.com/tracel-ai/burn/issues/1061) is asking for a `github action` yaml example to run build and release workflow, and I happened to do this in an `tch` feature enabled project, so I made this PR to provide an example. But now I found that it may not ask for an example...

As to the `CHANGELOG`, it's necessary to the release step in my github workflow example.